Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md

```diff
@@ -36,6 +38,22 @@ You can use the Microsoft Authenticator App for both iOS and Android or any othe
 
 ### Phone Authentication
 
-With the phone authenticator, when you log in to the Umbraco Cloud portal, you can choose to either receive an SMS with a code to enter when login in or a call to log you in.
+You have the option to use your phone when you log in to the Umbraco Cloud portal.
+You can choose to receive an SMS with a code or a call to log you in.
+
+:::note
+If you need to change your phone number.
+Make sure before deactivating the old number to change to the new number before the old number is disabled.
```

Comment:
Something doesn't sound quite right here 🤔

Reply:
Suggestion:
"Before deactivating your old phone number, please make sure to update the phone number used for your 2FA enrollment. Changing the phone number used for 2FA will require verification through the old number."